backend/mttnet/mttnet.go

```diff
@@ -242,7 +251,64 @@ func (n *Node) SiteClient(ctx context.Context, pid peer.ID) (WebsiteClient, erro
 	if err != nil {
 		return nil, err
 	}
-	return groups.NewWebsiteClient(conn), nil
+	return groups_proto.NewWebsiteClient(conn), nil
+}
+
+// GatewayClient opens a connection with the remote production gateway.
+func (n *Node) GatewayClient(ctx context.Context, url string) (cli GatewayClient, err error) {
```

Comment:
I'm not sure this belongs here in mttnet package, but if it works I'm OK. It also seems to be very similar to the code we have in the groups API, so maybe at some point we should unify this process of resolving web URLs to peer addresses.

I do have a question though: does this support *both* URLs and multiaddresses? Maybe we should just ignore the multiaddresses support, because no one will actually use it.

Reply:
It does support both, and You suggested it [here](https://github.com/MintterHypermedia/mintter/pull/1561#discussion_r1445330078) !:) I don't think it harms anybody to leave it, but no attachments to it. 
As for the `GatewayClient` they are not quite the same, since the SiteClient is an interface with more functions, but the gateway only has the push. It works ok and it isolates to call group functions outside group scopes